core/src/test/java/nl/nn/adapterframework/stream/MessageTest.java

```diff
@@ -95,143 +93,156 @@ protected void testAsByteArray(Message adapter) throws IOException {
 	}
 	
 	protected void testToString(Message adapter, Class<?> clazz) {
+		testToString(adapter,clazz,null);
+	}
+	
+	protected void testToString(Message adapter, Class<?> clazz, Class<?> wrapperClass) {
 		String actual = adapter.toString();
-		assertThat(actual, startsWith(clazz.getSimpleName()));
-		assertEquals(adapter.asObject().getClass().getName(), clazz.getName());
+		assertEquals(Message.class.getName(), actual.substring(0, actual.indexOf("@")));
+		int i = actual.indexOf(": ") + 2;
+		assertEquals(clazz.getTypeName(), actual.substring(i).substring(0, actual.substring(i).indexOf(": ")));
+		if (wrapperClass == null) {
+			assertEquals(adapter.asObject().getClass().getName(), clazz.getName());
+		} else {
+			assertEquals(adapter.asObject().getClass().getName(), wrapperClass.getName());
+		}
 	}
 	
 	@Test
-	public void testStreamAsStream() throws Exception {
+	public void testInputStreamAsInputStream() throws Exception {
```

Comment:
Ik vind het een beetje stom dat overal 'Stream' vervangen is door 'InputStream', en dat ik dan nu moet zoeken wat er nog meer veranderd is

Reply:
Vond ik duidelijker omdat het verwijst naar het aanroepen van testAsInputStream en om onderscheid te maken tussen binary en character stream en om verwarring te voorkomen dat je denkt dat StreamCaptured slaat op de aanroep van de capture method (alleen Captured slaat daarop). Wil je dat ik het terug verander?

Reply:
Nee, dat hoeft niet, ik ben er nu doorheen.